Cirrious.FluentLayout/AdvancedFluentLayoutExtensions.cs

```diff
@@ -77,7 +77,22 @@ public static IEnumerable<FluentLayout> FullHeightOf(this UIView view, UIView pa
 			yield return view.Bottom().EqualTo().BottomOf(parent).Minus(marginValue);
         }
 
-        public static IEnumerable<FluentLayout> VerticalStackPanelConstraints(this UIView parentView, Margins margins, params UIView[] views) =>
+		public static IEnumerable<FluentLayout> FullSizeOf(this UIView view, UIView parent, nfloat? margin = null)
+		{
+			return FullSizeOf(view, parent, new Margins((float)margin.GetValueOrDefault(DefaultMargin)));
+		}
+
+		public static IEnumerable<FluentLayout> FullSizeOf(this UIView view, UIView parent, Margins margins)
```

Comment:
FullHeightOf and FullWidthOf don´t support left/right or top/bottom margins. Just a single parameter. That´s why I can´t use them.